src/components/sidebar/routePathView/LinkNodeListViewTab.tsx

```diff
@@ -0,0 +1,134 @@
+            return 'Pysäkki';
+        }
+        case NodeType.CROSSROAD: {
+            return 'Risteys';
+        }
+        case NodeType.DISABLED: {
+            return (
+                <span className={s.notInUse}>
+                    Ei käytössä
+                </span>
+            );
+        }
+        default: {
+            return '-';
+        }
+        }
+    }
+
+    public getContent = () => {
+        const routePathLinks = this.props.routePath.routePathLinks;
+        if (!routePathLinks) return;
+        const sortedRoutePathLinks = routePathLinks.sort((a: IRoutePathLink, b: IRoutePathLink) => {
+            const keyA = a.orderNumber;
+            const keyB = b.orderNumber;
+            if (keyA < keyB) return -1;
+            if (keyA > keyB) return 1;
+            return 0;
+        });
+
+        const divs = sortedRoutePathLinks.map((link) => {
+            return (
+                <div
+                    key={link.orderNumber}
+                    className={s.linkNodeListViewItem}
+                >
+                    <div className={s.item}>
+                        <div className={s.attributes}>
+                            <div className={s.label}>
+                                Solmu:
+                                <div
+                                    className={s.id}
+                                >
+                                    {link.startNode.id}
+                                </div>
+                            </div>
+                            <div className={s.type}>
+                                Tyyppi: {this.getType(link.startNodeType)}
+                            </div>
+                            {
+                                link.isStartNodeTimeAlignmentStop &&
+                                <div className={s.timeAlignment}>
+                                    Ajantasauspysäkki
+                                </div>
+                            }
+                        </div>
+                        <div
+                            className={s.itemToggle}
+                        >
+                            <FaAngleRight />
+                        </div>
+                    </div>
+                    <div className={s.item}>
+                        <div className={s.attributes}>
+                            <div className={s.label}>
+                                Linkki:
+                                <div
+                                    className={s.id}
+                                    onClick={this.onClick}
+                                >
+                                    {link.id}
+                                </div>
+                            </div>
+                        </div>
+                        <div
+                            className={s.itemToggle}
+                        >
+                            <FaAngleRight />
+                        </div>
+                    </div>
```

Comment:
Duplicate HTML detected.

I would either make a list of objects [solmu, link, solmu, link, ...]
OR
renderItem(link);
renderItem(link.startNode);

And they would both use common method